README.md

```diff
@@ -30,33 +31,112 @@ global:
+* [`raw`](https://github.com/webpack/raw-loader): Loads raw content of a file (as utf-8)
+* [`val`](https://github.com/webpack/val-loader): Executes code as module and consider exports as JavaScript code
+* [`script`](https://github.com/webpack/script-loader): Executes a JavaScript file once in global context (like in script tag), requires are not parsed.
+
+**packaging**
+* [`file`](https://github.com/webpack/file-loader): Emits the file into the output folder and returns the (relative) url.
+* [`url`](https://github.com/webpack/url-loader): The url loader works like the file loader, but can return a Data Url if the file is smaller than a limit.
+* [`image`](https://github.com/tcoopman/image-webpack-loader): Compresses your images. Ideal to use together with `file` or `url`.
+* [`svgo-loader`](https://github.com/pozadi/svgo-loader): Compresses SVG images using [svgo](https://github.com/svg/svgo) library
+* [`baggage`](https://github.com/deepsweet/baggage-loader): Automatically require any resources related to the required one
+* [`polymer-loader`](https://github.com/JonDum/polymer-loader): Process HTML & CSS with preprocessor of choice and `require()` Web Components like first-class modules.
+
+**dialects**
+* [`coffee`](https://github.com/webpack/coffee-loader): Loads coffee-script like JavaScript
+* [`babel`](https://github.com/babel/babel-loader): Turn ES6 code into vanilla ES5 using [Babel](https://github.com/babel/babel).
+* [`livescript`](https://github.com/appedemic/livescript-loader): Loads LiveScript like JavaScript
+* [`sweetjs`](https://github.com/jlongster/sweetjs-loader): Use sweetjs macros. 
+* [`traceur`](https://github.com/jupl/traceur-loader): Use future JavaScript features with [Traceur](https://github.com/google/traceur-compiler).
+* [`typescript`](https://github.com/andreypopp/typescript-loader): Loads TypeScript like JavaScript.
+
+**templating**
+* [`html`](https://github.com/webpack/html-loader): Exports HTML as string, require references to static resources.
+* [`jade`](https://github.com/webpack/jade-loader): Loads jade template and returns a function
+* [`handlebars`](https://github.com/altano/handlebars-loader): Loads handlebars template and returns a function
+* [`ractive`](https://github.com/rstacruz/ractive-loader): Pre-compiles Ractive templates for interactive DOM manipulation
+* [`jsx`](https://github.com/petehunt/jsx-loader): Transform jsx code for [React](http://facebook.github.io/react/) to js code.
+* [`em`](https://github.com/yoshdog/emblem-loader): Compiles [Emblem](http://emblemjs.com/) to Handlebars.js
+* [`ejs`](https://github.com/okonet/ejs-loader): Loads EJS ([underscore](http://underscorejs.org/#template)( templating engine) template and returns a pre-compiled function
+* [`markdown`](https://github.com/peerigon/markdown-loader): Compiles Markdown to HTML
+* [`ng-cache`](https://github.com/teux/ng-cache-loader): Puts HTML partials in the Angular's $templateCache
+
+**styling**
```

Comment:
I think to highlight that webpack can produce `css` assets we can add [extract-text-webpack-plugin](https://github.com/webpack/extract-text-webpack-plugin) here.